libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -512,17 +512,10 @@ void AP_Baro::update(void)
 
     if (!_hil_mode) {
         for (uint8_t i=0; i<_num_drivers; i++) {
-            drivers[i]->update();
+            drivers[i]->backend_update(i);
```

Comment:
I like this new term "backend_update". We should use that terminology in all of the libraries!